src/Hedgehog.Xunit/InternalLogic.fs

```diff
@@ -219,16 +189,8 @@ module private PropertyBuilder =
             else
                 testMethod
 
-        try
-            methodToInvoke.Invoke(testClassInstance, args |> Array.ofList)
-        with
-        | :? TargetInvocationException as tie when not (isNull tie.InnerException) ->
```

Comment:
This was moved to the call site in `InternalLogic.fs`. Is this intentional? The unwrapping is no longer recursive here. If a `TargetInvocationException` wraps another `TargetInvocationException`, it won't fully unwrap anymore.